fiona/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: fiona
+  version: 1.6.2.dev1
+
+source:
+  git_tag: master
```

Comment:
If you are using master only because of `gdal 2.0.0`>  If so let's pin gdal to `< 2.0.0`.  We must avoid as much as we can using master/dev releases.  that brings all sort of instabilities and crashes for other packages that depends on it.